FreeRTOS/FreeRTOS-Plus-TCP
# IPv6/multi: Update IP header length after removing IP-options

Description
-----------
FreeRTOS+TCP doesn't support IP-options. While skipping over (and removing) the IP options, the IP-header fields are updated to reflect the same. However, the total length was not being updated. This PR fixes that issue.

Test Steps
-----------
Send a packet with IP-options (best to send an ICMP packet).
Receive the ICMP response which will have incorrect IP packet length.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
